llm_toolkit/models.py

```diff
@@ -61,6 +61,8 @@ class LLM:
 
   _max_attempts = 5  # Maximum number of attempts to get prediction response
 
+  temperature_range: list[float] = [0.0, 2.0]  # Default model temperature range
```

Comment:
Let's keep this attribute but make the value a constant  like `TEMPERATURE` defined at the top of the file:
```python
temperature_range: list[float] = TEMPERATURE_RANGE
```